talos:
  head_tilt_controller:
    type: effort_controllers/JointPositionController
    joint: updown_head_joint
    pid: {p: 500.0, i: 10.0, d: 1.0}

  head_pan_controller:
    type: effort_controllers/JointPositionController
    joint: leftright_head_joint
    pid: {p: 500.0, i: 10.0, d: 1.0}

